components/constants/webui_url_constants.cc

```diff
@@ -71,3 +71,6 @@ const char kSpeedreaderPanelHost[] = "brave-speedreader.top-chrome";
 const char kShortcutsURL[] = "chrome://settings/system/shortcuts";
 const char kChatUIURL[] = "chrome-untrusted://chat/";
 const char kChatUIHost[] = "chat";
+const char kNewTabPageHost[] = "newtab";
+const char kHistoryPageHost[] = "history";
+const char kBookmarksPageHost[] = "bookmarks";
```

Comment:
I don't think we should add these hosts here. This file is for Brave-only hosts/urls.

These hosts should just be declared in `chromium_src/components/sessions/content/content_serialized_navigation_driver.cc`.

Reply:
Ok. deleted.